### PR TITLE
move 2612 to Review

### PR DESCRIPTION
**Pull request by wschwab**
This PR moves EIP-2612 to Review. EIP-2612 enjoys widespread adoption, and imho should be moved forward in the EIP process. I have spoked with the EIP's author, @MrChico , and has told me that he is content with the specification as-is, and would be interested and it seeing it move forward.

### CODE REVIEW

**Review by lightclient — CHANGES_REQUESTED** [review]
Before moving to review, we should update the following:
* remove the Simple Summary and replace with a description in the preamble
* rename Implementation section to Reference Implementation
* remove the [1] from underneath the copyright section

**Comment by wschwab** [comment]
@lightclient I've pushed a commit that implements your suggestions

**Review by MrChico — APPROVED** [review]

**Comment by MrChico** [comment]
Thanks for picking this up @wschwab , this looks good to me!

File: EIPS/eip-2612.md [hunk 1]
```diff
@@ -1,36 +1,29 @@
...
-
-- transactions involving ERC-20 operations can be paid using the token itself rather than ETH,
-- approve and pull operations can happen in a single transaction instead of two consecutive transactions,
-
-while adding as little as possible over the existing ERC-20 standard.
-
 ## Abstract
-Arguably one of the main reasons for the success of ERC-20 tokens lies in the interplay between `approve` and `transferFrom`, 
-which allows for tokens to not only be transferred between externally owned accounts (EOA), but to be used in other contracts under application specific conditions by abstracting away `msg.sender` as the defining mechanism for token access control.
+Arguably one of the main reasons for the success of ERC-20 tokens lies in the interplay between `approve` and `transferFrom`, which allows for tokens to not only be transferred between externally owned accounts (EOA), but to be used in other contracts under application specific conditions by abstracting away `msg.sender` as the defining mechanism for token access control.
```
Comment (axic) [inline EIPS/eip-2612.md:15]: At least the first reference to each EIP should be linked:

```suggestion
Arguably one of the main reasons for the success of [ERC-20](./eip-20.md) tokens lies in the interplay between `approve` and `transferFrom`, which allows for tokens to not only be transferred between externally owned accounts (EOA), but to be used in other contracts under application specific conditions by abstracting away `msg.sender` as the defining mechanism for token access control.
```

File: EIPS/eip-2612.md [hunk 2]
```diff
@@ -1,36 +1,29 @@
...
 
 ## Motivation
-While ERC-20 tokens have become ubiquotous in the Ethereum ecosystem, their status remains that of second class tokens from the perspective of the protocol. The ability for users to interact with Ethereum without holding any ETH has been a [long outstanding goal](https://github.com/ethereum/EIPs/blob/ed621645c8f3bc5756492f327cda015f35d9f8da/EIPS/eip-101.md) and the [subject](./eip-1077.md) [of](./eip-777.md) [many](https://github.com/ethereum/EIPs/issues/1776#) [EIPs](https://eips.ethereum.org/EIPS/eip-1271).
+While ERC-20 tokens have become ubiquitous in the Ethereum ecosystem, their status remains that of second class tokens from the perspective of the protocol. The ability for users to interact with Ethereum without holding any ETH has been a [long outstanding goal](https://github.com/ethereum/EIPs/blob/ed621645c8f3bc5756492f327cda015f35d9f8da/EIPS/eip-101.md) and the [subject](./eip-1077.md) [of](./eip-777.md) [many](https://github.com/ethereum/EIPs/issues/1776#) [EIPs](https://eips.ethereum.org/EIPS/eip-1271).
```
Comment (axic) [inline EIPS/eip-2612.md:26]:
```suggestion
While ERC-20 tokens have become ubiquitous in the Ethereum ecosystem, their status remains that of second class tokens from the perspective of the protocol. The ability for users to interact with Ethereum without holding any ETH has been a [long outstanding goal](https://github.com/ethereum/EIPs/blob/ed621645c8f3bc5756492f327cda015f35d9f8da/EIPS/eip-101.md) and the [subject](./eip-1077.md) [of](./eip-777.md) [many](https://github.com/ethereum/EIPs/issues/1776#) [EIPs](./1271.md).
```

Comment (axic) [reply EIPS/eip-2612.md:26]: Why is 101 linked as a hash?

File: EIPS/eip-2612.md (hunk 1 above)
Comment (wschwab) [reply EIPS/eip-2612.md]: updated

File: EIPS/eip-2612.md (hunk 2 above)
Comment (wschwab) [reply EIPS/eip-2612.md]: updated (removed hash, made all links relative)

File: EIPS/eip-2612.md [hunk 3]
```diff
@@ -1,36 +1,29 @@
 ---
 eip: 2612
 title: permit – 712-signed approvals
+description: ERC20 approvals via secp256k1 signatures
```
[the remark this answers is not in the document]
Comment (wschwab) [reply EIPS/eip-2612.md]: fixed

**Review by lightclient — APPROVED** [review]

**Pull request merged**